.github/workflows/build-npm-packages.yml

```diff
@@ -104,8 +104,8 @@ jobs:
         working-directory: ice/js
 
       - name: Update Version
-        run: |
-          sed -i 's/"version":.*/"version": "${{ inputs.ice_version }}"/' package.json
+        working-directory: ice/js
+        run: npm version ${{ inputs.ice_version }}
```

Comment:
The sed command was breaking the JSON file, better use `npm version` to update the version.